tests/storage/storage_migration/test_mtc_storage_class_migration.py

```
@@ -126,3 +133,78 @@ def test_vm_storage_class_migration_b_to_a_with_running_and_stopped_vms(
+        "target_storage_class, online_vms_for_storage_class_migration",
+        [
+            pytest.param(
+                {"target_storage_class": py_config[STORAGE_CLASS_B]},
+                {"online_vm": [True]},  # Desired VM Running status for VMs in "vms_fixtures" list
+                marks=pytest.mark.polarion("CNV-11496"),
+                id="storage_migration_a_to_b_volume_hotplug_vms",
+            )
+        ],
+        indirect=True,
+    )
+    def test_vm_storage_class_migration_with_hotplugged_volume(
```

Pylint (3.3.7)

[refactor] Too many arguments (10/5)

(R0913)